tests/handlers/test_directory.py

```diff
@@ -394,30 +393,24 @@ class TestCreateAliasACL(unittest.HomeserverTestCase):
 
     servlets = [directory.register_servlets, room.register_servlets]
 
-    def prepare(self, reactor, clock, hs):
-        # We cheekily override the config to add custom alias creation rules
-        config = {}
+    def default_config(self):
+        config = super().default_config()
+
+        # We cheekily override the config to add custom alias creation and
+        # room publication rules.
```

Comment:
but you removed the room publication rules? Guess you changed this comment before invalidating it? :)

Reply:
Ah, yes. Good catch!